CMakeLists.txt

```diff
@@ -145,25 +139,20 @@ if(${VELOX_BUILD_TESTING})
   set(VELOX_ENABLE_TPCH_CONNECTOR ON)
   set(VELOX_ENABLE_SPARK_FUNCTIONS ON)
   set(VELOX_ENABLE_EXAMPLES ON)
+else()
+  set(VELOX_ENABLE_PARSE OFF)
 endif()
 
-if(${VELOX_ENABLE_EXAMPLES})
-  set(VELOX_ENABLE_EXPRESSION ON)
+if(VELOX_BUILD_TESTING OR VELOX_BUILD_TEST_UTILS)
+  set(cpr_SOURCE BUNDLED)
+  resolve_dependency(cpr)
+  set(VELOX_ENABLE_DUCKDB ON)
+else()
+  set(VELOX_ENABLE_DUCKDB OFF)
 endif()
 
-if(${VELOX_BUILD_BENCHMARKS})
```

Comment:
Thanks can you also remove it from here : https://github.com/facebookincubator/velox/blob/main/velox/docs/develop/TpchBenchmark.rst